src/cli/repl/commands/repl-commands.ts

```diff
@@ -52,7 +52,7 @@ export const helpCommand: ReplCommand = {
 	script:       false,
 	usageExample: ':help',
 	aliases:      [ 'h', '?' ],
-	fn:           output => {
+	fn:           (output) => {
```

Comment:
Was this your linters doing? in general i think we omit the parentheses for single-arg arrow syntax